frontend/workflows/projectSelector/src/card.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import type { ClutchError } from "@clutch-sh/core";
+import { Card as ClutchCard, CardHeader, Error } from "@clutch-sh/core";
+import { Grid } from "@material-ui/core";
+
+interface CardProps {
+  avatar?: React.ReactNode;
+  title?: React.ReactNode & string;
+  error?: ClutchError;
+  children: React.ReactNode;
+}
+
+const Card = ({ avatar, title, error, children }: CardProps) => (
```

Comment:
Lets add types to the params

Reply:
the param interface is above `CardProps` and we we type it here `{ avatar, title, error, children }: CardProps`

Reply:
Ah my bad this is a destructuring, i thought it was the return type lol.